### ISSUE TITLE
Problem with ipython console connecting to kernel

### ISSUE THREAD

**Issue by hkprivett** [ContinuumIO/anaconda-issues#932]
I have been having a problem with Spyder where upon startup (or anytime I try to open a new ipython consolein Spyder), I get a message in the ipython console that says "Connecting to kernel..." with a spinning wheel and eventually an error message that says "Python has stopped working". 
![capture1](https://cloud.githubusercontent.com/assets/13011517/17058253/5ad9c18a-4fed-11e6-8445-c003136fe29f.PNG)

I get the same error ("Python has stopped working") when I try to use "ipython console" or "jupyter console" from the command prompt.

I have tried the following:
1) Uninstalling and reinstalling Anaconda
2) Reset Spyder Settings
3) updating pyzmq (15.3.0)

Here is what I'm working with:
![capture](https://cloud.githubusercontent.com/assets/13011517/17057806/9a5dcd5e-4fea-11e6-82b5-e53dda6e0a0f.PNG)

Windows 7
Python 3.5.2 64 bit
Qt 4.8.7
PyQT4 4.11.4

Note: I have no problem running scripts from a regular python console in Spyder.

**Comment by ccordoba12** [comment]
There are some problems with IPython 5.0 that could be causing this.

**Comment by hkprivett** [comment]
Sorry! I didn't specify this before, but I  have ipython 4.2.0.  Do these problems you are referring to exist with ipython 4 as well?

Here are some more symptoms:  
1)  ipython runs fine when I type "ipython" on the command line:

![capture2a](https://cloud.githubusercontent.com/assets/13011517/17062785/6502fd7e-5002-11e6-94f2-74da1591b339.PNG)



2) when I try to open jupyter notebook from the command line, I the following errors:

![capture2b](https://cloud.githubusercontent.com/assets/13011517/17062813/7ca8b4f0-5002-11e6-9327-379f73efc074.PNG)

**Comment by ccordoba12** [comment]
> Do these problems you are referring to exist with ipython 4 as well?

No, they don't. I don't know what could be happening in this case.
